langchain/input.py

```diff
@@ -27,6 +27,11 @@ def get_colored_text(text: str, color: str) -> str:
     return f"\u001b[{color_str}m\033[1;3m{text}\u001b[0m"
 
 
+def get_bolded_text(text: str) -> str:
+    """Get bolded text."""
+    return f"\033[1m{text}\033[0m"
```

Comment:
Do we need to check if the current shell supports this? I don't know much about these codes....

Reply:
Same with the colored text! It's used in a lot of places in LC but will certainly look bad on an ascii terminal